integration-tests/src/test/scala/za/co/absa/spline/harvester/LineageHarvesterSpec.scala

```diff
@@ -338,10 +335,14 @@ object LineageHarvesterSpec extends Matchers {
       opActual.childIds should contain theSameElementsInOrderAs opExpected.childIds
       opActual.params should contain allElementsOf opExpected.params
 
-      opActual.schema shouldBe defined
-      val Some(actualSchema: Seq[UUID]) = opActual.schema
-      val Some(expectedSchema: Seq[UUID]) = opExpected.schema
-      actualSchema shouldReference actualAttributes as (expectedSchema references expectedAttributes)
+      for {
+        actualSchema <- opActual.schema.asInstanceOf[Option[Schema]]
+        expectedSchema <- opExpected.schema.asInstanceOf[Option[Schema]]
+      } {
+        actualSchema shouldReference actualAttributes as (expectedSchema references expectedAttributes)
+      }
+
+      opActual.schema.isDefined shouldBe opExpected.schema.isDefined
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of isDefined on optional type.](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4594774)